src/main/kotlin/no/nav/syfo/personstatus/api/v2/endpoints/PersonOversiktStatusApiV2.kt

```diff
@@ -31,8 +34,47 @@ fun Route.registerPersonoversiktApiV2(
     veilederTilgangskontrollClient: VeilederTilgangskontrollClient,
     personoversiktStatusService: PersonoversiktStatusService,
     personoversiktOppgaverService: PersonoversiktOppgaverService,
+    personoversiktSearchService: PersonoversiktSearchService,
 ) {
     route(personOversiktApiV2Path) {
+        post("/search") {
+            val callId = getCallId()
+            val token = getBearerHeader()
+                ?: throw java.lang.IllegalArgumentException("No Authorization header supplied")
+            val searchQuery = call.receive<SearchQueryDTO>().toSearchQuery()
```

Comment:
Tanken med `POST` og search query i body her er for å unngå semi-sensitive data i urlen, samt at det er enklest mtp validering og påkrevede felter.